README.md

```diff
@@ -17,94 +25,136 @@
 
 ### Chapter 2: Getting Started with Dash
 
- - Structure of a Dash app: data, layout, callback
- - Installation
+ - Structure of a Dash app
+ - Installation and setup
+ - Create your first Dash app
+ - Live updates and debugging
 
-### Chapter 3: Dash Components and Layout
+### Chapter 3: Dash Components and Layouts
 
- - Core components: dcc.dropdown, dcc.graph 
- - HTML components: html.div
  - Design the app layout
-
-### Chapter 4: Callback I- connecting two components
-
- - Callback Decorator: Input, Output, Component_id, Component_property
- - Callback function
+ - Linking a CSS file
```

Comment:
within this topic, should we discuss: "how to define your rows and columns with css file"

Reply:
I think we should do this with DCC and defer this advanced (annoying) topic to later.